src/main/java/com/bisnode/opa/TestRegoTask.java

```diff
@@ -1,35 +1,111 @@
 package com.bisnode.opa;
 
 import com.bisnode.opa.configuration.OpaPluginConvention;
+import com.bisnode.opa.junit.OpaToJunitConverter;
 import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.tasks.InputDirectory;
 import org.gradle.api.tasks.TaskAction;
+import org.gradle.tooling.TestExecutionException;
+import org.w3c.dom.Document;
 
+import javax.annotation.Nullable;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
 
 public class TestRegoTask extends DefaultTask {
 
+    public TestRegoTask() {
+        setGroup("opa");
```

Comment:
We should really decide when to use OPA and Rego :thinking:

Reply:
I'm torn on the other OPA/Rego decisions, but this one is pretty clear IMO - the plugin is called OPA Gradle plugin, so it makes more sense that the plugin group (grouping all tasks in IntelliJ et. al) is named "opa" rather than rego - this already contains tasks which are not related to Rego, such as starting and stopping the OPA server.